content/general/authentication.textile

```diff
@@ -30,7 +30,7 @@ Every application has one or more private API keys that you set up and "configur
 </a>
 
 **Using Token Authentication**
-Client-side devices should generally be considered untrusted, and as such, it is important that you minimize the impact of any credentials being compromised on those devices. The recommended strategy therefore for authenticating client-side devices is to issue them with time-limited tokens with only the privileges (capabilities) that are needed for that device. It is therefore the responsibility of the developer to identify each device before issuing a token. In the example below, we show you a a typical authentication workflow for client-side devices.
+Client-side devices should generally be considered untrusted, and as such, it is important that you minimize the impact of any credentials being compromised on those devices. Therefore, the recommended strategy for authenticating client-side devices is to issue them with time-limited tokens with only the privileges (capabilities) that are needed for that device. It is therefore the responsibility of the developer to identify each device before issuing a token. In the example below, we show you a a typical token authentication workflow for client-side devices.
```

Comment:
" we show you a a typical token " => "we show you a typical token". (not part of your changes but good to fix!)

Reply:
Good find, I missed it :) Thanks!